base/util/file.go

```diff
@@ -30,7 +30,7 @@ func EnsurePathWithinFilesDir(path, filesDir string) error {
 	if err != nil {
 		return err
 	}
-	if !strings.HasPrefix(absPath, absBase+string(filepath.Separator)) {
+	if strings.Compare(absPath, absBase) != 0 && !strings.HasPrefix(absPath, absBase+string(filepath.Separator)) {
```

Comment:
Logic looks okay, but is there a reason to use `strings.Compare()` instead of `!=`?

Reply:
There's no particular reason as such. I usually prefer to use a built-in function.